test/integration/tasks.test.js

```diff
@@ -80,6 +81,46 @@ describe("Tasks", function () {
     sinon.restore();
   });
 
+  describe("GET /tasks/overdue/current", function () {
```

Comment:
test 404 and 500 responses as well

Reply:
if im not wrong then 404 is for unauthorized users, right?
we already have tests for authentication and authorization middleware, it doesn't makes sense to write the tests to check for the same flow.

Reply:
> if im not wrong then 404 is for unauthorized users, right?

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404
I did not understand your comment here.